pkg/tkn/tkn.go

```diff
@@ -151,62 +142,6 @@ func (w *CapturingPassThroughWriter) Bytes() []byte {
 	return w.buf.Bytes()
 }
 
-func testCloser(closer io.Closer) {
```

Comment:
We are not using this anywhere?

Reply:
Nope, it's a private function which was only used in this file on line 168